source/NPC.cpp

```diff
@@ -500,12 +500,21 @@ bool NPC::HasSucceeded(const System *playerSystem, bool ignoreIfDespawnable) con
 					&& !(it->second & ShipEvent::ASSIST);
 			}
 			bool isHere = false;
+			bool isCarried = ship->CanBeCarried() && ship->GetParent();
```

Comment:
Shouldnt iscarried also check that the ship has no system, meaning that it is in facr in a carrier?
Right now this is more canBeCarried than actually isCarried, I think